assets/src/scss/base/_typography.scss

```diff
@@ -1,3 +1,5 @@
+@import "fonts";
+
```

Comment:
Couldn't we just import `_fonts` too in GF styles below? (like we do in the [main style file](https://github.com/greenpeace/planet4-master-theme/blob/master/assets/src/scss/style.scss#L20-L21))

Reply:
Yes good point